src/components/reminderMailer.js

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios'
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+import { ReminderButton } from './buttons'
+import { CardFooter } from './Card'
+import CloseModalCountdown from './closeModalCountdown'
+
+export function ReminderMailer({ item, borrower, contact, closeModal }) {
+  const [startCountdown, setStartCountdown] = useState(false)
+  const [state, setState] = useState({
+    name: borrower,
+    message: `Hallo ${borrower}, bitte denk daran, mir folgenden Gegenstand wieder
+    zurück zu geben: ${item}`,
+    email: contact,
+    sent: false,
+    buttonText: 'Erinnerung senden',
+  })
+
+  function onSend(event) {
+    event.preventDefault()
+
+    setState({
+      name: borrower,
+      message: `Hallo ${borrower}, bitte denk daran, mir folgenden Gegenstand wieder
+          zurück zu geben: ${item}`,
+      email: contact,
+      sent: false,
+      buttonText: '...wird gesendet',
+    })
+
+    let data = {
+      name: state.name,
+      email: state.email,
+      message: state.message,
+    }
+
+    axios
+      .post('/api/v1', data)
+      .then(res => {
```

Comment:
Maybe check the status code of res, you could receive a response that something has gone wrong on the server.